### PR TITLE
[AB#40278] ajv deduplication and mosaic depdencies update

### PR DESCRIPTION
**Pull request by AxTrusov**
# Pull Request

## Prerequisites

- [x] My code follows the coding conventions
- [x] All tests pass

## Description

- ajv dependency deduplication to fix build issues with updated @axinom/mosaic-cli
- mosaic dependencies updated to latest pre-release version
- updated message JSON schemas to work with latest asyncapi tooling

### CODE REVIEW

File: libs/media-messages/src/generated/schemas/payloads/media/commands/check-finish-ingest-item-command.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
   ],
   "properties": {
     "ingest_item_step_id": {
-      "description": "A unique identifier that is used to track ingest messages for a single document to keep ingest document status up-to-date.",
       "type": "string",
       "minLength": 1,
-      "pattern": "^$|.*\\S.*"
+      "pattern": "^$|.*\\S.*",
+      "description": "String of minimum length of 1 character, which is also cannot consist of only whitespace characters."
```
Comment (Zehelein) [inline libs/media-messages/src/generated/schemas/payloads/media/commands/check-finish-ingest-item-command.json]: A string with at least one character and not only whitespace characters.

File: libs/media-messages/src/generated/types/payloads/media/commands/check-finish-ingest-item-command.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 export interface CheckFinishIngestItemCommand {
   /**
-   * A unique identifier that is used to track ingest messages for a single document to keep ingest document status up-to-date.
+   * String of minimum length of 1 character, which is also cannot consist of only whitespace characters.
```
Comment (Zehelein) [inline libs/media-messages/src/generated/types/payloads/media/commands/check-finish-ingest-item-command.ts]: Here and further down we should keep the original message as it is much more helpful.

File: libs/media-messages/src/generated/types/payloads/media/commands/delete-entity-command.ts [hunk 3]
```diff
@@ -19,8 +18,5 @@ export interface DeleteEntityCommand {
    * The name of the entity type.
    */
   entity_type?: string;
-  /**
-   * Additional input data.
-   */
-  input?: BulkEntityInput;
+  input?: { [name: string]: unknown };
```
Comment (Zehelein) [inline libs/media-messages/src/generated/types/payloads/media/commands/delete-entity-command.ts:21]: The description got lost here and with the externalized type (and the description there) it was much clearer what this is used for.

**Review by EmperorRXF — APPROVED** [review]
@Zehelein, I just had a chat with @AxTrusov about this PR now. Think we both agree that the "documentation breaking change" is not ideal. At the same time, the code changes this PR is applying on media-template is quite important, as without them we leave the template in a broken state when do release the latest version.

So we're left with two choices rn, revert the libs changes we did on core-libs or go forward with this PR on media-template.

I now had a look at the original reason for us to bump the codegen libs, and it was due to a critical security alert we got via dependabot for possible code-injections (https://github.com/Axinom/Navy/security/dependabot/88)

So, considering all of this, think we can go forward with this PR - knowing that we have a follow-up task to address the breaking documentation change (soonish..). In the meantime, we can expect OSS users to find existing usages of the generated types already in code, and to some-degree derive the purpose of the individual fields of the types. In worse case, we can provide needed support if/when they reach out to us.

File: libs/media-messages/src/generated/types/payloads/media/commands/check-finish-ingest-item-command.ts (hunk 2 above)
Comment (AxTrusov) [reply libs/media-messages/src/generated/types/payloads/media/commands/check-finish-ingest-item-command.ts]: Related to Ruwans comment: https://github.com/Axinom/mosaic-media-template/pull/98#pullrequestreview-1500138990

File: libs/media-messages/src/generated/types/payloads/media/commands/delete-entity-command.ts (hunk 3 above)
Comment (AxTrusov) [reply libs/media-messages/src/generated/types/payloads/media/commands/delete-entity-command.ts:21]: Related to Ruwans comment: https://github.com/Axinom/mosaic-media-template/pull/98#pullrequestreview-1500138990

**Pull request merged**